src/test/java/com/kainos/ea/integration/JobRoleControllerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.kainos.ea.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.DropwizardWebServiceApplication;
+import org.kainos.ea.DropwizardWebServiceConfiguration;
+import org.kainos.ea.cli.JobRole;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRoleControllerTest {
+
+    static final DropwizardAppExtension<DropwizardWebServiceConfiguration> APP = new DropwizardAppExtension<>(
+            DropwizardWebServiceApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles() {
+        List<JobRole> response = APP.client().target("http://localhost:8080/api/job-roles")
```

Comment:
Can you test this on AWS/Github. It doesn't look like the test ran when the GitHub build